spec/models/organisation_spec.rb

```diff
@@ -242,6 +243,7 @@
 
   describe "scopes" do
     before do
+      described_class.destroy_all
```

Comment:
How come we're now doing this here?

Reply:
I should be able to remove this now. This was related to what I have been talking about that I keep getting organisations floating around in the DB that aren't meant to be there